Firebase/Messaging/FIRMessagingRemoteNotificationsProxy.m

```diff
@@ -587,7 +587,7 @@ id userInfoFromNotification(id notification) {
     // Cannot access the userInfo property.
     return nil;
   }
-  id notificationUserInfo = getNamedPropertyFromObject(notificationContent,
+  id notificationUserInfo = FIRMessagingPropertyNameFromObject(notificationContent,
                                                        @"userInfo",
                                                        [NSDictionary class]);
```

Comment:
Fix indents in this file too.

Reply:
Done. It's strange that script/style.sh didn't capture this.